src/pages/LandingPage.js

```diff
@@ -109,6 +94,7 @@ const LandingPage = () => {
       h(ButtonOutline, {
         style: { marginLeft: '2rem', padding: '1.5rem 1rem', textTransform: 'none' },
         onClick: () => {
+          Ajax().Metrics.captureEvent(`${Events.catalogView}:landingPageBanner`)
```

Comment:
Bt convention, this string should be added to the list in events.js instead of constructing an event name here.